phpBB/phpbb/db/migrator.php

```diff
@@ -742,7 +775,14 @@ public function migration_state($migration)
 	*/
 	protected function get_migration($name)
 	{
-		return new $name($this->config, $this->db, $this->db_tools, $this->phpbb_root_path, $this->php_ext, $this->table_prefix);
+		$migration = new $name($this->config, $this->db, $this->db_tools, $this->phpbb_root_path, $this->php_ext, $this->table_prefix);
+
+		if ($migration instanceof ContainerAwareInterface)
```

Comment:
want to fix the one migration that uses phpbb\content_visibility incorrectly atm?